tests/smoke/hello_test.go

```diff
@@ -21,8 +21,8 @@ func TestHelloBundle(t *testing.T) {
 	defer test.Close()
 	require.NoError(t, err, "test setup failed")
 
-	// Set up a fresh docker environment
-	mgx.Must(shx.RunV("docker", "system", "prune", "--all", "-f"))
+	// make sure the referenced image is not in local image cache
+	mgx.Must(shx.RunV("docker", "rmi", "carolynvs/whalesayd"))
```

Comment:
If the image is not in the cache, this line will fail. In cases where you just need to do cleanup before a test, I suggest not checking the error message (kind of like calling rmdir and then ignoring the error if the directory doesn't exist).